<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdminWindow</class>
 <widget class="QWidget" name="AdminWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <widget class="QLabel" name="labelTitle">
   <property name="geometry">
    <rect>
     <x>67</x>
     <y>35</y>
     <width>661</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(53, 53, 53);
font: 700 26pt &quot;Segoe UI&quot;;</string>
   </property>
   <property name="text">
    <string>Admin Panel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QTableWidget" name="tablePatients">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>170</y>
     <width>671</width>
     <height>371</height>
    </rect>
   </property>
   <property name="baseSize">
    <size>
     <width>150</width>
     <height>0</height>
    </size>
   </property>
   <attribute name="horizontalHeaderCascadingSectionResizes">
    <bool>true</bool>
   </attribute>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>326</number>
   </attribute>
   <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
    <bool>false</bool>
   </attribute>
   <column>
    <property name="text">
     <string>Name</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Age</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="btnLogout">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>560</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Logout</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>139</y>
     <width>671</width>
     <height>26</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="btnAdd">
      <property name="text">
       <string>Add</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnUpdate">
      <property name="text">
       <string>Update</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnDelete">
      <property name="text">
       <string>Delete</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLineEdit" name="txtName">
   <property name="geometry">
    <rect>
     <x>61</x>
     <y>110</y>
     <width>491</width>
     <height>24</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>nama pasien</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinAge">
   <property name="geometry">
    <rect>
     <x>559</x>
     <y>110</y>
     <width>171</width>
     <height>25</height>
    </rect>
   </property>
   <property name="minimum">
    <number>0</number>
   </property>
   <property name="maximum">
    <number>150</number>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>0</y>
     <width>821</width>
     <height>601</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image:url(:/user/image/admin.jpg)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>label</zorder>
  <zorder>labelTitle</zorder>
  <zorder>tablePatients</zorder>
  <zorder>btnLogout</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>txtName</zorder>
  <zorder>spinAge</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
